docs/docs/data/create.tsx

```diff
@@ -324,16 +324,110 @@ const createEndpointTableData = [
-  },
-  {
-    "name": "disabledFeaturesExclude",
-    "required": false,
-    "type": "String",
-    "description": (<>List (comma-separated) of features to no longer disable in a particular meeting. This is particularly useful if you disabled a list of features on a per-server basis but want to allow one of two of these features for a specific meeting. (added 2.6.9)<br /><br />The available options to exclude are exactly the same as for <code className="language-plaintext highlighter-rouge">disabledFeatures</code></>)
-  },
-  {
-    "name": "preUploadedPresentationOverrideDefault",
+    "description": (
+        <>
+          List (comma-separated) of features to disable in a particular meeting. (added 2.5)
```

Comment:
```suggestion
          List (comma-separated) of features to disable in a particular meeting.
```

With many of the options here being introduced later on, it's confusing what which portion was added in BBB 2.5. (based on a recent piece of feedback)